.env-devel

```diff
@@ -94,6 +94,10 @@ RABBIT_SECURE=false
 REDIS_HOST=redis
 REDIS_PORT=6379
 
+RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED=1
```

Comment:
Same here. please keep env vars sorted alphabetically. You can keep in groups though

Reply:
thanks, my mistake didn't notice the REGISTRY :)